randovania/games/samus_returns/logic_database/header.txt

```diff
@@ -434,7 +434,27 @@ Dock Weaknesses
       Lock type: FRONT_BLAST_BACK_IF_MATCHING
           Lay Power Bomb
 
-  > Dock Rando: Disabled
+  > Dock Rando:
+      Unlocked: Power Beam Door
+      Locked: Access Closed
+      Change from:
+          Charge Beam Door
+          Missile Door
+          Plasma Beam Door
+          Power Beam Door
+          Power Bomb Door
+          Spazer Beam Door
+          Super Missile Door
+          Wave Beam Door
+      Change to:
```

Comment:
not possible to change to access closed / permanently closed doors?

Reply:
Closed only exists with something else on the other side. A `doorclosedclosed` doesn't exist and have to be first implemented in the patcher.